app/src/main/java/com/infomaniak/drive/ui/fileList/SearchFragment.kt

```diff
@@ -191,23 +192,16 @@ class SearchFragment : FileListFragment() {
     private fun setToolbarListener() {
         toolbar.setOnMenuItemClickListener { menuItem ->
             if (menuItem.itemId == R.id.selectFilters) {
```

Comment:
🤔 Otherwise we can also do like this 😅
```kotlin
            (menuItem.itemId == R.id.selectFilters).also { 
                if (it) safeNavigate(R.id.searchFiltersFragment)
            }
```